run-unittests

```diff
@@ -46,10 +52,11 @@ echo ""
 echo ""
 echo "+++ Agent util-scripts Unit Tests +++"
 echo ""
+tail -n 9999999 -f /var/tmp/agent.util-scripts.out --pid $tpid &
```

Comment:
You don't worry about any other tail process *except* for the server scripts tail process which you wait for. Is that necessary? And if it is, why is it not necessary for all the other tails?

Reply:
Resolved this removing duplicate code, making a `tail_and_wait` function to do the same thing, use it for all the unit tests that are run, and finally, stop using a background tail and just let it run in the foreground.